app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::BooksController < ApplicationController
+  before_action :load_book, only: :show
+
+  def index
+    @books = Book.includes(:author).order("name ASC")
```

Comment:
```suggestion
    books = Book.includes(:author).order(:name)
```
asc là mặc định em nhé
nên viết scope nè

ngoài ra, books ni ko cần là instance variable đâu em

Reply:
dạ vâng c